pyodide_build/tests/recipe/test_bash_runner.py

```diff
@@ -53,11 +53,17 @@ def test_subprocess_with_shared_env_logging(capfd, tmp_path):
         dir.mkdir()
         p.run("echo 1000", script_name="test script", cwd=dir)
         cap = capfd.readouterr()
-        assert [l.strip() for l in cap.out.splitlines()] == [
-            "Running test script in",
-            str(dir),
-            "1000",
-        ]
+
+        # Clean and join all outputs to handle potential line breaks,
+        # and check for path components, instead of the exact path.
+        assert "Running test script in" in cap.out
+
+        # Normalize whitespace for path comparison
+        normalized_output = "".join(cap.out.split())
+        normalized_path = "".join(str(dir).split())
+        assert normalized_path in normalized_output
+
+        assert "1000" in cap.out
```

Comment:
Unfortunately, macOS runs with `zsh` and not `bash`, which forces us to resort to things like these. Changing to `bash` does not help, either.

A similar observation was noted in #105.

Reply:
Ah, I haven't explored why it doesn't help; it's just that I encounter the same failure.